modules/bootstrapped/src/generated/smithy4s/example/AudioEnum.scala

```diff
@@ -5,13 +5,12 @@ import smithy4s.Hints
 import smithy4s.Schema
 import smithy4s.ShapeId
 import smithy4s.ShapeTag
-import smithy4s.schema.EnumTag
-import smithy4s.schema.Schema.enumeration
+import smithy4s.schema.Schema.stringEnumeration
 
-sealed abstract class AudioEnum(_value: String, _name: String, _intValue: Int, _hints: Hints) extends Enumeration.Value {
```

Comment:
I'm fine with the rename, but the reordering seems a little tricky, especially because both values have type `String` and compilation would not fail

Reply:
for the most part, this stuff is meant to be code-generated, we only handcraft some of them for unit tests